packages/cli/src/layouts/page.html

```diff
@@ -1,7 +0,0 @@
-<html>
```

Comment:
not sure why the diff is only showing _page.html_ being removed, as the entire _layouts/_ directory was deleted?  🤔 
https://github.com/ProjectEvergreen/greenwood/tree/enhancement/issue-1251-remove-default-layout-and-pages-content/packages/cli/src